app/src/androidTest/java/org/mozilla/fenix/ui/HomeScreenTest.kt

```diff
@@ -299,6 +299,7 @@ class HomeScreenTest {
         }
     }
 
+    @Ignore("failing, see https://github.com/mozilla-mobile/fenix/issues/28472")
```

Comment:
```suggestion
    @Ignore("failing after a design refactor, see https://github.com/mozilla-mobile/fenix/issues/28472")
```

There's a nonzero chance we need to add a `testtag` parameter via `Modifier.semantics` to this as a follow-up. (See: `RecentlyVisited.RecentlyVisitedHistoryHighlight`)

Reply:
added additional context